lib/mixins/element-mixin.html

```diff
@@ -673,7 +678,18 @@
        * @override
        */
       attributeChangedCallback(name, old, value) {
-        if (old !== value) {
+        // process `disable-upgrade` specially
+        if (name === DISABLED) {
+          const disabled = value !== null;
+          if (this.__upgradeFlush && !disabled) {
+            this._flushProperties = this.__upgradeFlush;
+            this.__upgradeFlush = null;
+            this._flushProperties();
+          } else if (disabled) {
+            this.__upgradeFlush = this._flushProperties;
+            this._flushProperties = function() {};
```

Comment:
So this will kind of just pause property flushing if you apply `disable-upgrade` a second time... Don't think it's particularly problematic, just noting (we made it a one-shot in 1.x, but not sure that's particularly bad). Hmm, it may be problematic from a `splices` path notification point of view, now that I think about it...